assignment/assignment-api/api/src/java/org/sakaiproject/assignment/cover/AssignmentService.java

```diff
@@ -866,4 +866,12 @@ public static String escapeInvalidCharsEntry(java.lang.String param0) {
 			return null;
 		return service.escapeInvalidCharsEntry(param0);
 	}
+	
+	
+	public static Integer getScaleFactor() {
+		org.sakaiproject.assignment.api.AssignmentService service = getInstance();
+		if (service == null)
```

Comment:
Sakai adheres to Sun standards, and as such opening and closing braces are required even for one line if/else statements. I know it's visually appealing, but without the braces it can lead to problems down the road.